.github/workflows/triage.yml

```
@@ -24,11 +24,14 @@ jobs:
           ref: develop # DO NOT CHANGE
 
       - uses: actions/setup-python@a26af69be951a213d495a4c3e4e4022e16d87065
+      - uses: ./.github/actions/checkout-spack
```

It may be good to explicitly configure this to point at the `spack/spack-packages@develop` rather than do the clone inside of the scripts.

---

I agree it'd be good to be more explicit here, but we can't always clone develop since the latest Spack might not be compatible with the packages repository for a short time when we modify the API. This existing action reads the max Spack version from `.ci/`. I'd like to change this later on to use `spack/setup-spack` but we'll need to extract the commit from the config file and we'll need to create a new release of the action that supports v1.0 repositories.

---

Right, the part I am talking about is for the packages repo, not spack. I agree we should not be grabbing the head of spack/spack develop.

This line where
```
${{ inputs.path }}/bin/spack repo set --destination "$REPOSITORY_PATH" --scope site builtin
```

For this check we should point at the develop of packages, not the PR, when querying maintainers. This ensures that people can't remove a maintainer without them at least getting a ping.

Then change woiuld be
```suggestion
      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
        with:
            ref: develop
            fetch-depth: 1
            path: develop/
      - uses: ./.github/actions/checkout-spack
        with:
            builtin: develop/
```

---

What you're describing already happens in lines 22-24.

---

Oh duh, yea okay then I am fine on this point :)